CHANGELOG.md

```diff
@@ -50,6 +50,12 @@
   [#231](https://github.com/Kong/go-kong/pull/231)
 - Add possibility to client to send declarative configs via `ReloadDeclarativeRawConfig()`
   [#252](https://github.com/Kong/go-kong/pull/252)
+<<<<<<< HEAD
+- Add `RootBaseURL()` to Client
+=======
+- Add `RootBaseURL()` to Client #255
```

Comment:
looks like there is a misnaming here, as the method is called `BaseRootURL`, not `RootBaseURL`

Reply:
Thanks for staying vigilant!